### PR TITLE
Documentation for bouncie integration

### PR DESCRIPTION
**Pull request by mandarons**
## Proposed change

Documentation for the a integration - bouncie

## Type of change

Documentation for a new integration - bouncie
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/83247 
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3909
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/bouncie.markdown [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ha_config_flow: true
+ha_codeowners:
+  - '@mandarons'
+ha_domain: bouncie
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The `bouncie` integration uses the [Bouncie](https://www.bouncie.dev/) web service
```
Comment (frenck) [inline source/_integrations/bouncie.markdown]:
```suggestion
The Bouncie integration uses the [Bouncie](https://www.bouncie.dev/) web service
```

File: source/_integrations/bouncie.markdown [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+as a source for monitoring your car(s).
+
+## Setup
```
Comment (frenck) [inline source/_integrations/bouncie.markdown]:
```suggestion
## Prerequisites
```

File: source/_integrations/bouncie.markdown [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+To generate an Bouncie application credentials -
+
+1. Go to the [Bouncie Developer Page](https://bouncie.dev/login/) page
+2. Login with your `bouncie.com` credentials
+3. Click on `+ ADD APPLICATION` 
+4. Provide a `name` for application e.g. Home Assistant Integration
```
Comment (frenck) [inline source/_integrations/bouncie.markdown]:
```suggestion
4. Provide a `name` for the application e.g., Home Assistant Integration
```

File: source/_integrations/bouncie.markdown [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+5. Provide a unique `client_id` e.g. alphanumeric random string
```
Comment (frenck) [inline source/_integrations/bouncie.markdown]:
```suggestion
5. Provide a unique `client_id` e.g., an alphanumeric random string
```

File: source/_integrations/bouncie.markdown [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+6. Provide redirect URL to be your Home Assistant URL
+7. Click on `SAVE`
+8. Visit https://auth.bouncie.com/dialog/authorize?response_type=code&`client_id`=&redirect_uri=`redirect_url` in a browser.
+9.  Follow the prompts to provide your application access to your account
+10. Copy the alphanumeric string after `<redirect URL provided above>?code=...` - this is your `authorization_code`
```
Comment (frenck) [inline source/_integrations/bouncie.markdown:33]: This should not be needed? The Home Assistant OAuth flow can just take care of that?

File: source/_integrations/bouncie.markdown [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+## Setup
+
+To generate an Bouncie application credentials -
+
+1. Go to the [Bouncie Developer Page](https://bouncie.dev/login/) page
+2. Login with your `bouncie.com` credentials
+3. Click on `+ ADD APPLICATION` 
+4. Provide a `name` for application e.g. Home Assistant Integration
+5. Provide a unique `client_id` e.g. alphanumeric random string
+6. Provide redirect URL to be your Home Assistant URL
```
Comment (frenck) [inline source/_integrations/bouncie.markdown:29]: The redirect URL should use the My Home Assistant service we provide. See, for example, the Spotify integration as an example.

File: source/_integrations/bouncie.markdown [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    1.  Client ID 
+    2.  Client Secret
+    3.  Redirect URL
+    4.  Authorization code
+
+{% include integrations/config_flow.md %}
+
+<div class="note">
+This integration currently uses REST API only with polling.
+</div>
```
Comment (frenck) [inline source/_integrations/bouncie.markdown:44]: Not sure why this is a note or relevant? Is there a specific reason for this addition?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/bouncie.markdown (hunk 5 above)
Comment (mandarons) [reply source/_integrations/bouncie.markdown:33]: I agree but I couldn't figure out HA's OAuth flow. Do you have any example I can refer to?

File: source/_integrations/bouncie.markdown (hunk 7 above)
Comment (mandarons) [reply source/_integrations/bouncie.markdown:44]: Bouncie has Webhook API as well. This is just to inform user that data provided by this integration is through cloud polling using REST API.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**